packages/opentelemetry/src/sampler.ts

```diff
@@ -5,18 +5,21 @@ import { TraceState } from '@opentelemetry/core';
 import type { Sampler, SamplingResult } from '@opentelemetry/sdk-trace-base';
 import { SamplingDecision } from '@opentelemetry/sdk-trace-base';
 import {
-  SEMANTIC_ATTRIBUTE_HTTP_REQUEST_METHOD,
   SEMANTIC_ATTRIBUTE_SENTRY_OP,
   SEMANTIC_ATTRIBUTE_SENTRY_SAMPLE_RATE,
-  SEMANTIC_ATTRIBUTE_URL_FULL,
```

Comment:
I can't totally remove `SEMANTIC_ATTRIBUTE_URL_FULL` from core export because non-otel SDKs also use it, but maybe we should just hardcode the string?

Reply:
I have no strong opinions either way!